guides/release/components/helper-functions.md

```diff
@@ -302,70 +299,114 @@ Class helpers are useful when the helper logic is fairly complicated, requires
 fine-grained control of the helper lifecycle, is _stateful_ (we'll be
 discussing state in the next chapter), or requiring access to a [service](../../services/).
 
-## Built-in Helpers
+## Built-in and Common Helpers
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        Unlike nearly everything else in  template tag (`.gts`) files, these helpers do not need to be explicitly imported.
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="">
+  </div>
+</div>
 
 Below you will find some useful template helpers documented.
-For the full list of available helpers, you can check the [template helpers API documentation](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/).
+For the full list of available helpers, you can check the [template helpers API](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/) and [@ember/helpers](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper) documentation.
 
 ### The `get` helper
```

Comment:
given the above says 

>  do not need to be explicitly imported.

it's a little weird that we immediately import a non-keyword helper